tilavarauspalvelu/api/graphql/types/reservation/serializers/create_serializers.py

```diff
@@ -236,14 +236,14 @@ def check_max_reservations_per_user(self, user: AnyUser, reservation_unit: Reser
         if reservation_unit.max_reservations_per_user is None:
             return
 
-        existing_reservations_count = (
+        num_active_user_reservations = (
```

Comment:
This should probably be a reservation queryset method, so that changing it's logic doesn't ned to be done to two places.

Reply:
Valid point.
Moved the main part to `ReservationQuerySet`, but as both places that use the function need to do something different to the queryset it's not quite as clean as I'd have liked, but still better than before :)